bin/pot-to-php.js

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env node
```

Comment:
Should we run this script in `./bin/build-plugin-zip.sh`?

Reply:
I swear I commented before your commit :P

Reply:
Yeah, just realized that myself. I have added it to there.